cinema_service/settings.py

```diff
@@ -47,6 +42,7 @@
     "debug_toolbar",
     "cinema",
     "user",
+    "drf_spectacular",
```

Comment:
did you connect simple jwt library?

Reply:
Yea, but forgot to add to installed apps